polaris-core/src/main/java/org/apache/polaris/core/persistence/resolver/Resolver.java

```diff
@@ -43,8 +43,8 @@
 import org.apache.polaris.core.entity.PolarisPrivilege;
 import org.apache.polaris.core.persistence.PolarisMetaStoreManager;
 import org.apache.polaris.core.persistence.ResolvedPolarisEntity;
-import org.apache.polaris.core.persistence.cache.EntityCache;
 import org.apache.polaris.core.persistence.cache.EntityCacheByNameKey;
+import org.apache.polaris.core.persistence.cache.EntityCache;
```

Comment:
Looks like a spurious change actually broke the lexical ordering

Reply:
Ran spotless which seemed to have fixed it